src/main/resources/mapping.json

```diff
@@ -36,9 +60,26 @@
           "description": "XPath to the data for the field to map"
         },
         "translation": {
-          "enum" : ["toDouble", "toInteger", "lookupMaterialTypeId", "lookupLocationId", "lookupVendorId", "getPurchaseOptionCode"],
+          "enum" : [
+            "toDouble",
+            "toInteger",
+            "lookupMaterialTypeId",
+            "lookupLocationId",
+            "lookupVendorId",
+            "getPurchaseOptionCode",
+            "lookupWorkflowStatusId",
+            "lookupFundId",
+            "toDate",
+            "lookupReceiptStatusId",
+            "lookupPaymentStatusId",
+            "lookupActivationStatusId"
+          ],
           "description": "Method to translate the specified type"
         },
+        "translateDefault": {
```

Comment:
Is it being used in the code? There are mappings that have this value set to **true** but I don't see how it affects the translation when the mapping is evaluated.

Reply:
Yes. You can see how it is used in DataSource.java.

Reply:
I think the primary use case is for lookups... e.g. you provide some literal value in the default, and want/need to do a lookup using that value.